Cargo.toml

```diff
@@ -13,11 +13,15 @@ description = """
 Generates an ebuild for a package using the in-tree eclasses.
 """
 
-[badges]
-travis-ci = { repository = "cardoe/cargo-ebuild" }
+[badges.travis-ci]
+repository = "cardoe/cargo-bitbake"
```

Comment:
This can't be right. I think I have an idea where you copied this from... ;)

Reply:
yep